scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -828,6 +830,44 @@
       (log/error t "In" pool-name "pool, error launching tasks for"
                  (cc/compute-cluster-name compute-cluster) "compute cluster"))))
 
+(defn filter-matches-for-ratelimit
+  "Given a set of matches, determine which compute clusters are beyond the rate limit and filter matches in those compute clusters out."
+  [matches]
+  (let [augmented-matches (->> matches
+                               (group-by match->compute-cluster)
+                               (map
+                                 (fn [[compute-cluster matches-in-compute-cluster]]
+                                   (let [compute-cluster (or compute-cluster )
```

Comment:
?

Reply:
Fixed. Leftover wart from trying to work around it not being easy to get the compute cluster name.